### ISSUE TITLE
Example with Zero-Shot Classification gives: Task zero-shot-classification is not supported for the ONNX Runtime pipeline. Supported tasks are []

### ISSUE THREAD

**Issue by mkalisz77** [davidberenstein1957/classy-classification#49]
Hi, 
Below example from main page doesn't work. Is it something wrong with this code or with my Python/Spacy installation. Sorry for bother you. I'm not an expert yet in ML/Spacy/Pipelines.

```
import spacy

data = ["furniture", "kitchen"]

nlp = spacy.blank("en")
nlp.add_pipe(
    "classy_classification",
    config={
        "data": data,
        "model": "typeform/distilbert-base-uncased-mnli",
        "cat_type": "zero",
        "device": "gpu"
    }
)

print(nlp("I am looking for kitchen appliances.")._.cats)

# Output:
#
# [{"furniture": 0.21}, {"kitchen": 0.79}]
```

**Comment by davidberenstein1957** [comment]
Hi @mkalisz77 could you provide some more errors and stack traces?

**Comment by davidberenstein1957** [comment]
This should be solved in version 1.0.1

**Comment by mkalisz77** [comment]
Works, as expected. 
Thank you.
